CSS/Redes Sociales/css/style.css

```diff
@@ -0,0 +1,84 @@
+/*!
+ * All Rights Reserved
+ * This software is proprietary information of
+ * Intelligent Sense
+ * Use is subject to license terms.
+ * Filename: style.scss
+ */
+/*
+ * Author:      cviquez@intelligentsense.com
+ * Date:        03/04/2019
+ * Description: Main COMPASS file. Import all the partials you need here.
+ */
+/**
+* Mixin for image replacement
+*/
+/* ==========================
+    Column main definition
+=============================*/
+
+.main-container {
+  margin: 25px;
+  border: 25px;
+  padding: 25px;
+  content: 25px;
+}
+
+.column  {
+    position: relative;
+    float: left;
+	width: 20%;
+	padding: 10px;
+    height: 300px;
+    margin: 50px;
+}
+
+/* ==========================================================================
+   Independent Columns
+   ========================================================================== */
+.facebook  {
+    font-family: 'Courier New', Courier, monospace;
+    background-color: rgba(0, 0, 255, 0.815);
+
+}
+
+.twitter  {
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    background-color: rgb(0, 174, 255); 
+}
+
+
+.instagram  {
+    font-family: 'Nunito', sans-serif;  
+    background-color: rgba(165, 77, 42, 0.815); 
+}
+
+
+.tittle {
+    content: " " ;
```

Comment:
What is this content about?

Reply:
When I was searching I saw this example and I tried to do it the the same way, then I realized that it was so much easier with an <h1></h1> on html. And I forgot it. But actually I don't Know the correct usage.
![Screen Shot 2019-04-05 at 1 56 31 PM](https://user-images.githubusercontent.com/48696020/55653131-c56fb500-57aa-11e9-89c5-97b65971ac52.png)